Cargo.toml

```diff
@@ -31,6 +31,9 @@ thiserror = "1.0"
 regex = "1"
 radix_trie = "0.2.1"
 const_format = "0.2.31"
+chrono = { version = "0.4.38", default-features = false, features = ["alloc", "std"] }
+cel-interpreter = { git = "https://github.com/clarkmcc/cel-rust", branch = "master" }
+cel-parser = { git = "https://github.com/clarkmcc/cel-rust.git", branch = "master" }
```

Comment:
We need something newer than `v0.8.0`, as we need this [fix](https://github.com/clarkmcc/cel-rust/issues/62)

Reply:
Last week was released [v0.8.1](https://github.com/clarkmcc/cel-rust/releases/tag/v0.8.1) which includes a bunch of your fixes :)